packages/core/src/__tests__/smoke.e2e.test.js

```diff
@@ -11,17 +11,17 @@ describe('CMS Design System smoke test', () => {
       .setRect(1024, 768, 0, 0);
   });
 
-  it('Should click Learn how to get started and open startup/installation/', async() => {
+  it('Should click Learn how to get started and open for-developers/installation/', async() => {
```

Comment:
I'd rather we not rename this to "for developers" -- we've had use cases in the past where designers or content people have needed the information contained here and this label can be off-putting.